kimchi/src/circuits/constraints.rs

```diff
@@ -668,23 +669,28 @@ impl<F: PrimeField + SquareRootField> Builder<F> {
                     },
                 )
                 .sum();
-            for runtime_table in runtime_tables.iter() {
-                num_lookups += runtime_table.len();
+            // After that on the runtime tables
+            if runtime_tables.is_some() {
+                let runtime_tables = runtime_tables.as_ref().unwrap();
```

Comment:
Can we apply the `if let Some(x) = runtime_tables` construct here? (it would be consistent with the code later on L682

Reply:
See 58c9b33a7bde95f0cda4e06fc2a35b1dda752fc2